### PR TITLE
agent: fix request merging when comment is missing

### PR DESCRIPTION
**Pull request by dpausp**
Don't fail merging when the first request doesn't have an explicity set comment. Also put newlines between merged comments.

PL-131698

@flyingcircusio/release-managers

## Release process

Impact:

Changelog:

* (not relevant)

## Security implications

- [x] [Security requirements](https://wiki.flyingcircus.io/System_Development_Guideline#Security_requirement_principles_and_testing) defined? (WHERE)
  - nothing new, just a fix for manually-created maintenance requests. 
- [x] Security requirements tested? (EVIDENCE)
  - checked merging on test system, Python tests cover merging requests now

### CODE REVIEW

**Review by osnyx — APPROVED** [review]
I heard that tests are pending as well, so LGTM

**Comment by dpausp** [comment]
I pushed a change after the approval but it's just the same behaviour as for comments applied to estimates. Also added tests which cover the changes.

**Pull request merged**
